website/docs/r/bigtable_gc_policy.html.markdown

```diff
@@ -0,0 +1,79 @@
+---
+layout: "google"
+page_title: "Google: google_bigtable_gc_policy"
+sidebar_current: "docs-google-bigtable-gc-policy"
+description: |-
+  Creates a Google Cloud Bigtable GC Policy inside a family.
+---
+
+# google_bigtable_gc_policy
+
+Creates a Google Cloud Bigtable GC Policy inside a family. For more information see
+[the official documentation](https://cloud.google.com/bigtable/) and
+[API](https://cloud.google.com/bigtable/docs/go/reference).
+
+
+## Example Usage
+
+```hcl
+resource "google_bigtable_instance" "instance" {
+  name         = "tf-instance"
+  cluster_id   = "tf-instance-cluster"
+  zone         = "us-central1-b"
+  num_nodes    = 3
+  storage_type = "HDD"
+}
+
+resource "google_bigtable_table" "table" {
+  name          = "tf-table"
+  instance_name = "${google_bigtable_instance.instance.name}"
+  
+  column_family {
+    family = "name"
+  }
+}
+
+resource "google_bigtable_gc_policy" "policy" {
+  instance_name = "${google_bigtable_instance.instance.name}"
+  table         = "${google_bigtable_table.table.name}"
+  family        = "name"
+  
+  max_age {
```

Comment:
Do you mind showing an example requiring `UNION` or `INTERSECTION`?

Reply:
I've added example for UNION, it's good for you?